python/smashbox/utilities/oc_utils.py

```diff
@@ -0,0 +1,49 @@
+from smashbox.script import config
+
+import smashbox.utilities 

```

Comment:
Changing this to `from smashbox.utilities import runcmd` seems to make it import "runcmd", but then "runcmd" for some reason cannot find the global logger...

Also it seems that the global logger is not available in this module, even though it's available in __init__ during a test run. Maybe the multi processing engine normally creates the logger instance somehow...

I still don't understand it. It's too convoluted.